### ISSUE TITLE
cfcatch variables

### ISSUE THREAD

**Issue by bendur** [foundeo/cfdocs#1638]
I was thinking it would be useful to have a list of cfcatch variables somewhere. 

Perhaps on the cfcatch tag page, and/or the error handling guide (or somewhere else entirely?). 

For reference, Adobe has a [dedicated page under coldfusion variables](https://helpx.adobe.com/coldfusion/cfml-reference/reserved-words-and-variables/coldfusion-tag-specific-variables/cfcatch-variables.html) and also [lists them on the cfcatch reference page](https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-c/cfcatch.html).

**Comment by JCOvergaar** [comment]
agreed - this information is arguably more useful than some of the tags attributes.
